packages/aws-fargate/images/test-images/utils

```diff
@@ -11,7 +11,7 @@ function normalizeArgs {
     instana_layer_without_tag=instana-aws-fargate-nodejs-local
     INSTANA_LAYER_MODE=local
   elif [[ $instanaLayerMode = aws ]]; then
-    instana_layer_without_tag=410797082306.dkr.ecr.us-east-2.amazonaws.com/instana-aws-fargate-nodejs
+    instana_layer_without_tag=767398002385.dkr.ecr.us-east-2.amazonaws.com/instana-aws-fargate-nodejs
```

Comment:
suggestion: Store the aws id at least somewhere centralized for sh scripts.

Reply:
created the card INSTA-15396. I will pick it up in Q4.